tt_metal/hw/firmware/src/brisc.cc

```diff
@@ -308,11 +316,13 @@ int main() {
         {
             DeviceZoneScopedMainN("BRISC-FW");
 
+#ifdef NCRISC_HAS_IRAM
```

Comment:
Would be cleaner to put this in a sep function call but there was a comment about saving a branch so I kept the ifdef here

Reply:
you can remove the ifdefs and rely on the ifdefs in the routine so this turns into a nop